tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
 
 envlist =
-    flake8, twistedchecker-diff
+    flake8
 
     # Twisted 15.5 is the first version to support Python 3 (3.3+)
-    trial-py{27,py,34,35,36}-tw{155,166,current,trunk}
+    coverage-py{27,py,34,35,36}-tw{155,166,current,trunk}
+
+    coverage_combine
```

Comment:
Including this in the `envlist` is bad because that means it gets run in some random order in both `tox` with no arguments and `detox` with no arguments (where it's actively harmful).  You can still run it with `tox -e` without putting it in the envlist.